releasenotes/notes/add-manifest-validation-e4bbe4b1b04dca78.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
```

Comment:
I don't think this PR needs changelog or QA since it is just adding tests. There should be no user-facing impact. Please remove this and add labels no-changelog and qa/no-code-change

Reply:
so I can remove this yaml file?